site/security.html

```diff
@@ -135,13 +135,13 @@ <h2>Reporting a vulnerability</h2>
           <ol>
             <li>Demand status updates on existing submissions. We will do our best to be timely.</li>
             <li>Ask for ransom, or any suggested bounty. Don't be that guy.</li>
-            <li>Send a link to a best practice unless you can demonstrate workable exploit that would be solved by linked content.</li>
+            <li>Send a link to a best practice unless you can demonstrate a workable exploit that would be solved by the linked content.</li>
             <li>Describe a theoretical vulnerability, we deal mostly in the real world.</li>
-            <li>Disclose a vulnerability found in a 3rd party tools or product. If you have identified a vulnerability in a 3rd party product that we are using, please disclose this to the product owner first. We would still appreciate you letting us know, but we cannot offer you a bounty in this scenario.</li>
+            <li>Disclose a vulnerability found in a 3rd party tool or product. If you have identified a vulnerability in a 3rd party product that we are using, please disclose this to the product owner first. We would still appreciate you letting us know, but we cannot offer you a bounty in this scenario.</li>
           </ol>
           <p>It may take us from 1-2 weeks to review the report and verify the vulnerability. If your report not does not meet the submission guidelines, or does not adequately prove a vulnerability the report will be rejected.</p>
           <p>In most cases, when a report is rejected, the FusionAuth security team will respond to ask for additional clarification or evidence. However, if it is clear that the reporter has not invested any reasonable amount of time to follow the submission guidelines, no response will be offered. We value your time and effort, we ask that you do the same for us.</p>
-          <p>Once we determine a report has identified a real vulnerability you can expect that we will let you know we have accepted your report. If we are able to offer you a bounty, we will communicate the amount, and ask you for preferred payments. Please note that we do not guarantee we can use your preferred method, but we will do our best.</p>
+          <p>Once we determine a report has identified a real vulnerability you can expect that we will let you know we have accepted your report. If we are able to offer you a bounty, we will communicate the amount, and ask you for preferred payment method(s). Please note that does not guarantee we can use your preferred method(s), but we will do our best.</p>
```

Comment:
I don't have a strong opinion here, but I tend to just decide if this is singular or plural and then not use the `(s)`. So we could just say "you preferred payment method". I don't know that it is important to use a plural here.

Reply:
I thought I saw other places where we use `(s)`, but I also don't have a strong opinion other that not mixing singular and plural, so I'll update to singular.